test/executor.test.js

```diff
@@ -271,6 +271,62 @@ describe('executor', function() {
         done();
       });
     });
+
+    describe ('for mixins', function() {
+      it('defines `mixins` files', function() {
+        var example = appdir.writeFileSync('mixins/example.js',
+          'module.exports = ' +
+          'function(app, options) { app.mixinExample = true; }');
```

Comment:
Since the mixin is never called, you can export an empty function. Also note that the first function arg is `Model`, not `app`.

```js
module.exports = function(Model, options) {};
```

Please apply the same change for `mixins/time-stamps.js` too.